### PR TITLE
IGVF-389-software

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: components/search/__tests__/list-renderer.test.js [hunk 1]
```diff
@@ -1220,3 +1222,115 @@ describe("Test the MeasurementSet component", () => {
...
+    const title = screen.getByTestId("search-list-item-title");
+    expect(title).toHaveTextContent(/^Bowtie2 2.4.4$/);
+
+    const meta = screen.getByTestId("search-list-item-meta");
+    expect(meta).toHaveTextContent("J. Michael Cherry, Stanford");
+
+    const status = screen.getByTestId("search-list-item-status");
+    expect(status).toHaveTextContent("released");
+
+    const paths = MeasurementSet.getAccessoryDataPaths([item]);
```
Comment (forresttanaka) [inline components/search/__tests__/list-renderer.test.js:1300]: Change this to `SoftwareVersion.getAccessoryDataPaths([item])` and you’ll get full Jest coverage.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Nice new implementation of the software-version table! I hope the SortableGrid component was easy to figure out how to configure.

File: components/software-version-table.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    title: "Download",
+    display: ({ source }) => {
+      return (
+        <Link
+          href={source.downloaded_url}
+          key={source.downloaded_url}
+          target="_blank"
+        >
+          {source.downloaded_url}
+        </Link>
```
[the remark this answers is not in the document]
Comment (mingjiecn) [reply components/software-version-table.js:41]: Really helpful! Thank you!

**Review by forresttanaka — APPROVED** [review]
😎

**Pull request merged**